nuxt/pages/about.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="about">
-    <hero-banner :background="require('../../assets/shrine.jpg')" class="mb-4">
+    <hero-banner :background="require('../static/about/shrine.jpg')" class="mb-4">
```

Comment:
You should just be able to reference this like so:

```suggestion
    <hero-banner background="/backgrounds/shrine.jpg" class="mb-4">
```

Also, I think the file should be stored in `/static/backgrounds`.